docs/src/pages/component-demos/dialogs/ConfirmationDialog.js

```diff
@@ -6,13 +6,8 @@ import PropTypes from 'prop-types';
 import { createStyleSheet } from 'jss-theme-reactor';
 import customPropTypes from 'material-ui/utils/customPropTypes';
 import Button from 'material-ui/Button';
-import { List, ListItem, ListItemText } from 'material-ui/List';
-import {
-  Dialog,
-  DialogContent,
-  DialogActions,
-  DialogTitle,
-} from 'material-ui/Dialog';
+import List, { ListItem, ListItemText } from 'material-ui/List';
```

Comment:
Duplication and consistency.  One would expect `/List` to `export default List`, whether it be an index or a file itself.  To have it also be a named export might be a convenience, but it is inconsistent with other libraries.